video/avi_decoder.cpp

```diff
@@ -201,6 +201,7 @@ bool AVIDecoder::parseNextChunk() {
 	case ID_STRH:
 		handleStreamHeader(size);
 		break;
+	case ID_HDRL: // Header list.. what's it doing here? Probably ok to ignore?
```

Comment:
Why is this a question?

Reply:
Ah, because I'm honestly not sure of the answer - this pattern pops up in a few of the crusader videos (not all) and it didn't seem to affect anything to just skip.  If anyone else understands AVI containers better than me I'd be happy to get advice.